src/System.Private.CoreLib/shared/System/Globalization/CompareInfo.cs

```diff
@@ -173,7 +176,7 @@ private void OnDeserializing(StreamingContext ctx)
             m_name = null;
         }
 
-        void IDeserializationCallback.OnDeserialization(object sender)
+        void IDeserializationCallback.OnDeserialization(object? sender)
```

Comment:
I don't think this should be nullable

Reply:
AFAIR I couldn't change it because I'd have to change interface as well - possibly I'll pick up this interface next

Reply:
If the interface is not annotated, you should be able to change it. But anyway, I don't think this should be nullable, not sure though.

Reply:
ok, seems no issues when I changed it